tests/test_path_utils.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import os
+import sys
+
+sys.path.insert(0, '../src/asmcnc/core_UI')
+from path_utils import get_path
+
+class TestPathUtils(unittest.TestCase):
+
+    def test_get_path_files_only(self):
+        target_dir = "router_machine.py"
+        expected_result = "/easycut-smartbench/src/asmcnc/comms/router_machine.py"
+        result = get_path(target_dir, files_only=True).replace("\\", "/")
```

Comment:
the replace should not be needed. get_path and search_tree both take care.

Reply:
Fixed :)